lf/demo/demo_robot.lf

```diff
@@ -140,18 +177,24 @@ reactor Controller {
     
     reaction(bumped) {=
         // Logging sensor inputs...
+    // We can't do deadline :(
     =} deadline(10 msec) {=
         printf("Soft deadline missed. Waiting for the next input.\n");
     =}
 }
 
+// Here is the equivalent container in Luos for this reactor : https://github.com/Luos-io/Examples/tree/master/Projects/l0/Dc_motor/lib/DC_motor
+// The same one using models : https://github.com/Luos-io/Examples/tree/template/Drivers/DC_motor
+// As you can see here there is 2 containers representing left and right motors on the same folder. This is because there is one physical driver but 2 features called "DC_motor1_mod" and "DC_motor2_mod".
+// We also have some example generating containers depending on the number of pluged motors we have for example.
 reactor Motors {
     input _in:int[];
     state stopped:bool(false);
     reaction(_in) {=
         if (!self->stopped) {
         	printf("left motor: %d, right motor: %d\n", _in->value[0], _in->value[1]);   
         }
+    // We can't do that in Luos for now. This is THE missing feature.
```

Comment:
The deadline in LF is relative to the logical time tag of the trigger. If `current_physical_time - logical_time > deadline`, then there is a deadline violation and the handler will execute.

Reply:
We have the `current_physical_time` witch (for us) is basically the systick but we don't have the `logical_time`.